API.yaml

```diff
@@ -1871,31 +1871,47 @@ components:
           description: Tags for the entire line
           type: integer
           format: int32
-    ColumnHeaderEntry:
+    DataType:
       type: object
       properties:
-        id:
-          description: Unique id to identify this column in the backend.
-          type: integer
-          format: int32
-        name:
-          description: Displayed name for this column
-          type: string
-        description:
-          description: Description of the column
+        unit:
+          description: The units used for this data, to be appended to the data
           type: string
-        type:
-          description: Type of data associated to this column
+        dataType:
+          description: Type of data a value represents. Gives a hint on the formatting to give to the data
           type: string
+          enum:
+            - NUMBER
```

Comment:
Could you add some description on what each type means and can be used for? Also, if omitted it's a assumed a type string.